readme.md

```diff
@@ -107,6 +114,35 @@ Extension of the config file.
 
 You would usually not need this, but could be useful if you want to interact with a file with a custom file extension that can be associated with your app. These might be simple save/export/preference files that are intended to be shareable or saved outside of the app.
 
+#### migrations
+
+ type: `Object`<br>
+ Default: `undefined`
+
+ Migrations to be run between versions.
+
+ Useful for transitioning conf changes between application versions. Ex:
+ ```js
+ // version is set to 0.0.1, the current app version is 2.0.8, old is set to 1
+ const store = new Conf({
+ 	migrations: {
+ 		'0.0.0': store => {
+ 			store.set('bad key', 2);
+ 		},
+ 		'1.0.0': store => {
+ 			const old = store.get('old');
+ 			store.set('new', old);
+ 			store.delete('old');
+ 		},
+ 		'1.0.2': store => {
+ 			store.set('a new key', 't');
+ 		}
```

Comment:
I think it could be useful to have a special `*` migration that is run for every version. If you need more advanced handling.

Reply:
Think I might leave that as a separate PR if that's cool.